src/components/goalDetail/GoalListItem.tsx

```
@@ -1,27 +1,15 @@
 import { faFileLines, faEllipsisV } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { useEffect, useRef, useState } from 'react';
+import { useClickOutside } from '@/hooks/useClickOutside';
 
 type Props = {
   item: { id: number; task: string; date: string; done: boolean };
   onToggle: (id: number) => void;
```

ESLint

[error] 'id' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)